requirements-app.txt

```diff
@@ -63,19 +63,19 @@ PyJWT>=2.6.0
 # import errors with newer versions, but that package can be updated to version 1.1.0 if we upgrade Flask.
 PyPDF2 < 2.0.0
 
-python-dotenv>=0.21.0
+python-dotenv>=0.21.1
 python-magic>=0.4.27
 
 # jsonschema requires this package to validate the date-time format but doesn't include it as a dependency.
 # Validation fails silently without it.
 # https://python-jsonschema.readthedocs.io/en/latest/validate/?highlight=date-time#validating-formats
 rfc3339-validator>=0.1.4
 
-sentry-sdk[flask]>=1.11.1
+sentry-sdk[flask]>=1.15.0
 
 # 10 December 2022: Ignoring security vulnerability 51668 in Makefile.  It is fixed in >=2.0.0b1.
 SQLAlchemy < 1.4
 
-twilio>=7.15.4
+twilio>=7.16.2
```

Comment:
Just adding a comment in case people look in the future. The updated dependencies have been rolled back, so this update only includes updating the Future dependency from 0.18.2 to 0.18.3